Production_SOP.md

```diff
@@ -0,0 +1,71 @@
+# Core Operations Production SOP
+## UC Santa Cruz Genomics Institute, Computational Genomics Lab
+ 
+## Environment Setup
+ 
+ These steps should only have to be run once. Toil-box images will have to be updated periodically to stay current.
```

Comment:
So for readability you should always backquote strings that a user would type into a command line.

So for example:

```
Bash can be started by running the `bash` command.
```

Or in this case:

```
The `toil-box` image will …
```